admin/src/js/controllers/targets-edit.js

```diff
@@ -95,6 +95,18 @@ angular.module('controllers').controller('TargetsEditCtrl',
         });
     };
 
+    const isTargetUnique = (settings) => {
+      if ($scope.editing) {
+        // Here we are editing a target, assume it is unique 
+        return $q.resolve({ settings: settings, unique: true });
```

Comment:
We can't assume it's unique because we allow people to edit the ID. Maybe we should make the ID readonly when editing? Otherwise it'll be hard to check that it's unique...